parser/shared/src/main/scala/hmda/parser/fi/lar/LarCsvParser.scala

```diff
@@ -0,0 +1,98 @@
+package hmda.parser.fi.lar
+
+import hmda.model.fi.lar._
+
+object LarCsvParser {
+  def apply(s: String): LoanApplicationRegister = {
+    val values = s.split('|').map(_.trim)
+    val id = values(0).toInt
+    val respId = values(1)
+    val code = values(2).toInt
+    val loanId = values(3)
+    val loanDate = values(4)
+    val loanType = values(5).toInt
```

Comment:
Isn't this going to blow up the whole parser with the first non-Int string it comes across?